tpl/fmt/fmt_test.go

```diff
@@ -0,0 +1,14 @@
+package fmt
+
+import (
+	jww "github.com/spf13/jwalterweatherman"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestErrorf(t *testing.T) {
+	errorCount := jww.LogCountForLevel(jww.LevelError)
+	ns := New()
+	assert.EqualValues(t, ns.Errorf("%s\n", "This is an errorf test"), "This is an errorf test\n", "")
+	assert.EqualValues(t, errorCount+1, jww.LogCountForLevel(jww.LevelError), "Error log count not incremented")
```

Comment:
Error count should be 1 for this test. And you need to assert the correct logger.

Reply:
I'm a bit lost here. Is there another way I should be checking the log count? 
I'm unsure how I would apply https://github.com/spf13/jWalterWeatherman/blob/master/log_counter.go#L34 on the distinct logger only.

Reply:
Let's put it this way: The log count check you currently have does not make sense (it does not make sense). I'm not sure if the distinct logger exports its internal logger, so it may be better to remove this check than to check for something that does not test what you do.

Reply:
Ok. I've removed the test.